### ISSUE TITLE
Make wiki mirrors use doc mirrors for API docs (and others)

### ISSUE THREAD

**Issue by tfoote** [ros-infrastructure/mirror#12]
From: ros-infrastructure/roswiki#234 @gavanderhoorn 

As the reason to visit the mirrors is typically because the main site is down, would it make sense to somehow change the base URL for the wiki mirrors to point to the doc mirrors as well?

Example: make the 'Code API' link for [angles](http://mirror-eu.wiki.ros.org/angles) point to the [mirrored API docs](http://mirror-eu.docs.ros.org/lunar/api/angles/html), instead of the [regular ones](http://docs.ros.org/lunar/api/angles/html).

**Comment by tfoote** [comment]
Being self contained would be nice. Both for the case outlined and for the offline/non-networked case.

It's a challenge since writing a sed script to do the substitution would not be too hard to do the url substitution. But it will break the rsync caching for updates to the mirror forcing a reload of all the files.

**Comment by dirk-thomas** [comment]
The substitution could easily be reverted before doing the next rsync?

**Comment by gavanderhoorn** [comment]
My apologies for not opening the issue here.

I don't run a docs/wiki mirror, but from your comment @tfoote I understand it's all plain html? Or should the manifests be run through `sed` to update the urls?

**Comment by tfoote** [comment]
Yes, it's all plain html. You'd be running sed over the resultant html files.

The sed script to undo the remapping would work. There'd be a brief period during the resync when the links would be broken doing the sed reversion rsync and then rerunning sed, but that would be managable and also way better than the current state.
